src/physics/em/SeltzerBergerModel.cc

```diff
@@ -69,14 +69,13 @@ SeltzerBergerModel::SeltzerBergerModel(ModelId               id,
  */
 auto SeltzerBergerModel::applicability() const -> SetApplicability
 {
-    // TODO: these energy ranges need to be the intersection of SB table data
-    // and available cross sections.
-    CELER_NOT_IMPLEMENTED("SeltzerBergerModel::applicability");
+    // TODO: Do we need to load applicabilities, e.g. lower and upper, from
+    // tables?
 
     Applicability electron_applic;
     electron_applic.particle = this->host_pointers().ids.electron;
-    electron_applic.lower    = units::MevEnergy{1};
-    electron_applic.upper    = units::MevEnergy{1e5};
+    electron_applic.lower    = units::MevEnergy{10e-4};
+    electron_applic.upper    = units::MevEnergy{10e8};
```

Comment:
I think you mean `1e-4` and `1e8`? Those are the ranges that are coming from the `geant-data-cat` tool for our current standard physics set for brems.